plans/system-roles.fmf

```diff
@@ -5,22 +5,18 @@ environment:
   DOCKERFILE_SYSTEMD: Dockerfile.systemd
   KEYLIME_SERVER_ROLE_UPSTREAM_URL: "https://github.com/linux-system-roles/keylime_server.git"
   KEYLIME_SERVER_ROLE_UPSTREAM_BRANCH: main
+  NETAVARK_FW: nftables
```

Comment:
Why it's set here this var?? @kkaarreell

Reply:
1. Its needed for running rhel9 containers on c10s as there were some changes in the networking stack.